cabot/cabotapp/jenkins.py

```diff
@@ -30,10 +30,16 @@ def get_job_status(jenkins_config, jobname):
             raise Exception("job has no build")
         last_build = client.get_build_info(jobname, last_completed_build['number'])
 
+        if job['lastSuccessfulBuild']:
+            last_good_build_number = job['lastSuccessfulBuild']['number']
+        else:
+            last_good_build_number = 0
```

Comment:
How about:

```
last_good_build_number = job.get('lastSuccessfulBuild', {}).get('number', 0)
```

Reply:
Good call done :)

Since `last_good_build_number` is then only used once, shall we just change to 
```
ret['consecutive_failures'] = last_build['number'] - job.get('lastSuccessfulBuild', {}).get('number', 0)
```
or is that becoming too obfuscated ?

Reply:
Ah, we actually have a test-failure with this construct:
```
======================================================================
ERROR: test_job_no_good_build (cabot.cabotapp.tests.tests_jenkins.TestGetStatus)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/mock/mock.py", line 1305, in patched
    return func(*args, **keywargs)
  File "/code/cabot/cabotapp/tests/tests_jenkins.py", line 182, in test_job_no_good_build
    status = cabot_jenkins.get_job_status(self.mock_config, 'job-no-good-build')
  File "/code/cabot/cabotapp/jenkins.py", line 33, in get_job_status
    last_good_build_number = job.get('lastSuccessfulBuild', {}).get('number', 0)
AttributeError: 'NoneType' object has no attribute 'get'
```

Reply:
Ah that's right, `lastSuccessfulBuild` can still be set, but set to `None`. Great, haha 😅